numpy/core/_add_newdocs_scalars.py

```diff
@@ -257,3 +257,19 @@ def add_newdoc_for_scalar_type(obj, fixed_aliases, doc):
         >>> np.{ftype}(-.25).as_integer_ratio()
         (-1, 4)
         """.format(ftype=float_name)))
+
+# XXX Neat way to get all names. Same as TODO above.
+for int_name in ('ubyte', 'ushort', 'uint', 'ulonglong',
+                 'byte', 'short', 'int', 'long', 'longlong'):
+    add_newdoc('numpy.core.numerictypes', int_name, ('bit_count',
```

Comment:
How is the build failing here? It's passing for me locally.
> AttributeError: type object 'int' has no attribute 'bit_count'

Reply:
I think `numpy.core.numerictypes.int` is the Python `int`, which is not a NumPy type. You probably don't want to write to its docstring.

Reply:
Don't forget about `intc` and `uintc` as well.

Reply:
Oh nice, yeah @mattip ,
```py
>>> np.core.numerictypes.int
<class 'int'>
```

I guess it worked for me as I was using 3.10

Sure @BvB93 , will add it

Reply:
I have added to `numpy.int32`, so `intc` also gets it.